querybook/server/models/datadoc.py

```diff
@@ -117,6 +117,12 @@ class DataCell(Base):
         order_by="QueryExecution.id.desc()",
     )
 
+    comments = relationship(
+        "Comment",
+        secondary="data_cell_comment",
+        backref=backref("data_cell", uselist=False),
```

Comment:
should it be `uselist=True` as a datadoc can have multiple comments?

also I dont see this `comments` or `data_cell` is used anywhere.